src/test/java/uk/gov/hmcts/ccd/domain/service/callbacks/CallbackServiceTest.java

```diff
@@ -139,10 +144,43 @@ public void shouldTrackCallbackEventOnException() throws Exception {
 
     }
 
+    @Test
+    @DisplayName("Should LogAll callback event")
+    public void shouldLogAllCallbackEvent() throws Exception {
+        List<String> ccdCallbackLogControl = new ArrayList<String>();
+        ccdCallbackLogControl.add("*");
+        doReturn(ccdCallbackLogControl).when(applicationParams).getCcdCallbackLogControl();
+        callbackService.send(URL, CALLBACK_TYPE, caseEventDefinition, null, caseDetails, (Boolean)null);
+
+        verify(appinsights).trackCallbackEvent(eq(CALLBACK_TYPE), eq(URL), eq("200"), any(Duration.class));
```

Comment:
not sure I understand what we are verifying here

Reply:
updated the tests